src/resources/locales/de.json

```diff
@@ -84,7 +84,7 @@
         "deleteFeed": "Willst du diesen Katalog wirklich löschen?",
         "deletePublication": "Willst du diese Publikation wirklich löschen?",
         "import": "Willst du diese EPUB-Dateien wirklich importieren?",
-        "importError": "Ein Fehler ist aufgetreten, überprüfen Sie die Dateiendung (.epub / .lcpl / .audiobook).",
+        "importError": "Ein Fehler ist aufgetreten, überprüfen Sie die Dateiendung (.epub / .lcpl / .audiobook / .lpf).",
```

Comment:
With the `i18next` "translator" we can easily handle interpolation, so could we remove the duplication of file extensions with `acceptedExtensionArray.join(" / ")`?
https://www.i18next.com/translation-function/interpolation